kbase/narrative
# DATAUP-449 add view configure mode for bulk import cells

# Description of PR purpose/changes

Implements the "View Configure" tab that appears while jobs are running / completed.
Really, all that happens here is this creates a mode switch for the configure, file path, and params widgets. Default is "edit" mode, with "view" mode as an option when the flag is set. The only real difference is that "view" mode uses the `view` variant of the input widgets.

There's a couple of those widgets identified here that don't really play as expected. I suspect there are others, but these fixes fill the need at the moment. I'm sure there's be a bugfix jamboree at some point...

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-449

# Testing Instructions
* Details for how to test the PR:
* get a running bulk import cell, and click on "View Configure", this should show up with read only / disabled versions of all of the input widgets.

# Dev Checklist:
